aten/src/ATen/CMakeLists.txt

```diff
@@ -393,10 +397,13 @@ endif()
     ${native_sparse_hip_hip}
     ${native_quantized_hip_hip}
     ${native_transformers_hip_hip} ${native_transformers_src_hip_hip}
+    ${native_ck_hip}
+    ${native_ck_kernels_hip}
```

Comment:
The inclusion of these files can be made conditional on `USE_ROCM_CK_GEMM` introduced in https://github.com/pytorch/pytorch/pull/152951, so we don't have to explicitly exclude for Windows, since USE_ROCM_CK_GEMM will be disabled for Windows by definition.